content/implement/client.md

```diff
@@ -0,0 +1,209 @@
++++
+date = "2016-01-30T11:01:06-05:00"
+title = "Client"
+weight = 9
+
+[menu.main]
+name = "Client"
+identifier = "implement client"
+parent = "implement"
++++
+
+Once the API dsl is used to define types and resources, goagen automatically generates a
```

Comment:
Maybe this could be reworded as:

`goagen` can generate a Go client package from an API design. The package makes it possible to make requests programmatically to the service implemented using the same design. `goagen` generates a client command line tool which uses the generated package to make the actual requests. Finally `goagen` can also generate a JavaScript client library based on the [axios](ihttps://github.com/mzabriskie/axios) JavaScript library.